### DISCUSSION TITLE
Using this for my Nostr client

### DISCUSSION THREAD

**Discussion by kfrancis** [enisn/UraniumUI#179] [Show and tell]
Long time volo supporter/licensee, working on a .net nostr lib and maui client [Link](https://github.com/kfrancis/NuSocial)

Still a work in progress, having a hard time getting some of the uranium bits to work but I do what I can.

**Comment by enisn** [comment]
Feel free to ask questions on some use-cases

Also, check [documentation](https://enisn-projects.io/docs/en/uranium/latest) for basic usages of components. If you need to override and customize components, you can always check the [source code](https://github.com/enisn/UraniumUI/tree/develop/src) and contact when you stuck.

**Reply by kfrancis** [reply]
Right now, I'm just trying to recolor and it's not going well. Can't seem to get away from those built-in greens.

**Reply by enisn** [reply]
You can override Colors & Styles by the following document:
https://enisn-projects.io/docs/en/uranium/latest/themes/material/ColorsAndStyles
